{"event": "verbose", "uuid": "6624c085-bfc6-457f-a0f0-90815c114f5f", "counter": 6, "stdout": "\u001b[0;31mThe error appears to be in '/home/duchess/Documents/src/github.com/edwardtheharris/ansible-kcp/tmp_dir_sphinx_ansible/temp_file.yaml': line 1, column 3, but may\u001b[0m", "start_line": 5, "end_line": 6, "runner_ident": "6a4ce341-d7a2-4e66-a9cb-e0436f9c8701", "created": "2024-07-23T18:49:22.293062+00:00"}